### DISCUSSION TITLE
export 'useComponentDefaultProps' (imported as 'useComponentDefaultProps') was not found

### DISCUSSION THREAD

**Discussion by AbyerAli** [mantinedev/mantine#2358] [Q&A] [answered]
WARNING in ./node_modules/@mantine/rte/esm/components/RichTextEditor/RichTextEditor.js 70:13-37
export 'useComponentDefaultProps' (imported as 'useComponentDefaultProps') was not found in '@mantine/core' (possible exports: ACTION_ICON_SIZES, AVAILABLE_TRANSITIONS, AVATAR_SIZES, Accordion, AccordionItem, ActionIcon, Affix, Alert, AlphaSlider, Anchor, AppShell, Aside, AspectRatio, Autocomplete, Avatar, AvatarsGroup, BADGE_SIZES, BURGER_SIZES, BUTTON_SIZES, BackgroundImage, Badge, Blockquote, Box, Breadcrumbs ...

i have been getting this error on mantine Rich text editor, need some help pls how do i fix this.

**Comment by rtivital** [comment, marked as answer]
Hi @AbyerAli, you have `@mantine/` packages versions mismatch, make sure that all Mantine packages have the same version in package.json.
